terraform/staging_embargo_bucket.tf

```diff
@@ -0,0 +1,116 @@
+resource "aws_s3_bucket" "api_staging_embargo_dandisets_bucket" {
```

Comment:
Is it possible to make this a module, so you don't repeat yourself and (more importantly) ensure the configuration between staging and prod stays in sync?

Reply:
I think that would be possible. There are some differences between the different bucket configurations but I think they will be parameterizable. I will look into it.

Reply:
I created a `dandiset_bucket` module and used it for the new embargo buckets. We still need to update the non-embargoed sponsored and staging buckets to use the module as well, but that involves a lot of terraform state manipulation that I don't want to do right now so I'm filing an issue to deal with later.

Reply:
https://github.com/dandi/dandi-infrastructure/issues/107